IdentityCore/src/MSIDError.h

```diff
@@ -69,9 +69,49 @@ typedef NS_ENUM(NSInteger, MSIDErrorCode)
     MSIDErrorInvalidClient = -51015,
     MSIDErrorInvalidGrant = -51016,
     MSIDErrorInvalidParameter = -51017,
+    
+    /*!
+     The user or application failed to authenticate in the interactive flow.
+     Inspect MSALOAuthErrorKey and MSALErrorDescriptionKey in the userInfo
+     dictionary for more detailed information about the specific error.
+     */
+    MSIDErrorAuthorizationFailed = -52018,
+
+    /*!
+     The state returned by the server does not match the state that was sent to
+     the server at the beginning of the authorization attempt.
+     */
+    MSALErrorInvalidState = -52501,
+    /*!
+     Interaction required errors occur because of a wide variety of errors
+     returned by the authentication service.
+     */
+    MSIDErrorMismatchedUser             = -52101,
```

Comment:
just curious, when is this error returned?

Reply:
When parameter sets a user, and we force webview to login as a different user. 
We handled it as an error in MSAL,  we can discuss about this.

Reply:
nit: can we put errors in increasing order? (MSIDErrorInvalidState is in the middle)